doc/tasks/php_parser.md

```diff
@@ -0,0 +1,73 @@
+# Phpparser
+
+The Phpparser task will test your changes for blacklisted or whitelisted keywords, such as `die()`, `var_dump()` etc.
+It lives under the `php_parser` namespace and has following configurable parameters:
+
+```yaml
+# grumphp.yml
+parameters:
+    tasks:
+        php_parser:
+            visitors_options:
+              function_call_visitor:
+                blacklist: ["var_dump", "error_log"]
+                whitelist: ["count", "get_class"]
```

Comment:
`whitelist` was a try to distinct two groups but, you're right the task fail anyway, so it's not a nice try !